### PR TITLE
apollo-parser: add description support for SchemaDefinition

### LINKED ISSUE #185: apollo-parser: SchemaDefinition should support description
Regarding the spec of GraphQL, `SchemaDefinition` should support description in the parser https://spec.graphql.org/October2021/#sec-Schema

### PR DESCRIPTION
**Pull request by bnjjj**
close #185 

Add support of `Description` for `SchemaDefinition`. 
Also Add `StringValue` node in a `Description`

### CODE REVIEW

**Review by lrlna — CHANGES_REQUESTED** [review]
nice!!

while you're here, could I ask you to add a test for accessing description in a schema definition via the AST?

**Comment by bnjjj** [comment]
Done

File: crates/apollo-parser/src/parser/grammar/description.rs [hunk 1]
```diff
@@ -6,5 +6,45 @@ use crate::{Parser, SyntaxKind};
...
+#[cfg(test)]
+mod tests {
+    use crate::ast;
+
+    use super::*;
+    #[test]
+    fn it_can_access_arguments_in_fields() {
+        let schema = r#"
+"""
+What a dumb description
```
Comment (lrlna) [inline crates/apollo-parser/src/parser/grammar/description.rs:22]:
```suggestion
description for Query object type
```

File: crates/apollo-parser/src/parser/grammar/description.rs [hunk 2]
```diff
@@ -6,5 +6,45 @@ use crate::{Parser, SyntaxKind};
...
+    p.bump(SyntaxKind::STRING)
+}
+
+#[cfg(test)]
+mod tests {
+    use crate::ast;
+
+    use super::*;
+    #[test]
+    fn it_can_access_arguments_in_fields() {
```
Comment (lrlna) [inline crates/apollo-parser/src/parser/grammar/description.rs:19]:
```suggestion
    fn it_can_access_definition_description() {
```

File: crates/apollo-parser/src/parser/grammar/description.rs [hunk 3]
```diff
@@ -6,5 +6,45 @@ use crate::{Parser, SyntaxKind};
...
+        for definition in document.definitions() {
+            if let ast::Definition::ObjectTypeDefinition(obj_def) = definition {
+                assert_eq!(
+                    obj_def
+                        .description()
+                        .unwrap()
+                        .string_value()
+                        .unwrap()
+                        .to_string(),
+                    "\"\"\"\nWhat a dumb description\n\"\"\"\n"
```
Comment (lrlna) [inline crates/apollo-parser/src/parser/grammar/description.rs:43]: we have a really nifty `into` impl for string values that strips `"`.
```suggestion
                        .string_value()
                        .into(),
                    "description for Query object type"
```

**Review by lrlna — APPROVED** [review]

**Pull request merged**